test/pkg/environment/common/setup.go

```diff
@@ -50,6 +50,8 @@ var (
 		{First: &v1alpha5.Provisioner{}, Second: &v1alpha5.ProvisionerList{}},
 		{First: &v1.LimitRange{}, Second: &v1.LimitRangeList{}},
 		{First: &schedulingv1.PriorityClass{}, Second: &schedulingv1.PriorityClassList{}},
+	}
+	FinalizableObjects = []functional.Pair[client.Object, client.ObjectList]{
```

Comment:
Can you make a comment about why we decided to do this so we don't forget the reason down the line?